### DISCUSSION TITLE
아스키 아트 프로그램 요구사항

### DISCUSSION THREAD

**Discussion by Lcat12345** [MyeongWoonJang/cpp-study#178] [General]
![image](https://user-images.githubusercontent.com/62102830/221357792-16f09441-0368-491e-8c2e-2294ba5b20d2.png)

**Comment by Lcat12345** [comment]
다음은 제가 구상한 프로그램 프리뷰입니다.

![preview1](https://user-images.githubusercontent.com/62102830/221387245-b976c80f-716a-416b-b307-ad11efb2ce24.png)

애니메이션 아이콘 https://icon-library.com/icon/animation-icon-png-24.html

**Reply by Lcat12345** [reply]
[요구사항 명세서 엑셀파일](https://github.com/Woon-2/cpp-study/files/10831150/default.xlsx)

**Comment by Woon-2** [comment]
![냠](https://user-images.githubusercontent.com/73771162/221386294-659450f5-ad50-42a9-a1d8-fc1b585cd50e.PNG)

#### 이미지 출처
- 팔레트 아이콘: https://www.flaticon.com/free-icon/palette_2400597?term=color&page=1&position=12&origin=search&related_id=2400597   
- 디스켓 아이콘: https://www.flaticon.com/free-icon/diskette_907027?term=save&page=1&position=11&origin=search&related_id=907027
- 필름 아이콘: https://www.flaticon.com/free-icon/motion-graphic_4661514?term=animation&page=4&position=51&origin=search&related_id=4661514
- 알파벳 주사위 아이콘: https://www.flaticon.com/free-icon/block_496314?term=alphabet&page=1&position=29&origin=search&related_id=496314
- 팝업 아이콘: https://www.flaticon.com/free-icon/new-window_2704747?term=popup&page=1&position=19&origin=search&related_id=2704747
- 전구 아이콘: https://www.flaticon.com/free-icon/solution_9782496?related_id=9782496&origin=pack

**Comment by Hsg2311** [comment]
![UI](https://user-images.githubusercontent.com/75879925/221392961-17a1cc20-2f7b-4fa2-908b-e0378b88d619.png)
